### DISCUSSION TITLE
Urgent Issue: Overusage and Hobby Plan Suspension

### DISCUSSION THREAD

**Discussion by rajput-hemant** [vercel/community#4754] [Help]
### Summary

Dear Vercel Team,

I hope this message finds you well. I am writing to bring to your attention a critical issue that I have encountered with my Vercel project, "infinitunes." I am a student and have been relying on your generous free tier to host my project, and I am extremely grateful for the opportunity it has provided me.

Recently, I noticed a sudden and unexplained spike in the execution of my project's functions, which led to a significant increase in resource usage. About a day or two ago, my function execution time was approximately 5 GB-Hours, but within a very short span of time, it skyrocketed to over 300+ GB-Hours. Unfortunately, I wasn't able to check my email during that time due to nighttime in my region, and by the time I discovered the issue, my hobby plan was paused due to overusage.

My project, "infinitunes," relies on another project, "jio-saavn-api" which serves as an API server. I suspect that both of these projects may have contributed to the overusage, but without access to analytics, I cannot definitively determine whether the spike in resource consumption was due to an unexpected influx of requests from my site or if it was a result of any actions on Vercel's side.

I would like to stress that I am genuinely hooked to Vercel's services and I do not wish to migrate to another platform. As a student, I am currently unable to afford a Vercel subscription, and I kindly request your assistance in resolving this issue.

I understand that Vercel offers a fantastic free tier, and I am sincerely grateful for the support it has provided to countless developers like myself. I hope that you can investigate this situation and help me regain access to my hobby plan, as it is crucial for my ongoing academic and personal projects.

Your prompt attention to this matter is greatly appreciated, and I hope for a favorable resolution. If there are any steps I need to take on my end to prevent such overusage in the future, please do let me know. Thank you for your understanding and support.

Sincerely,

### Steps to Reproduce

none

![1699267575](https://github.com/vercel/community/assets/68769346/78b53308-3da4-4152-a7b6-0679ef26dff8)

**Comment by amyegan** [comment]
Hi @rajput-hemant. As a workaround, you can use a [Pro Trial](https://vercel.com/docs/accounts/plans/pro/trials#) to temporarily increase your limits. That will let you dig into what is causing the higher usage and get a sense for when your project's usage will be back within Hobby limits.

**Reply by amyegan** [reply]
The [Usage](https://vercel.com/docs/limits/usage) report in your Dashboard has some additional details that can help your investigation. If you are using [Web Analytics](https://vercel.com/docs/analytics#) in the project, that will have more info on unique visitors and page views. If the high usage problem is ongoing, rather than a spike, you may also find some helpful information in the [runtime logs](https://vercel.com/docs/observability/runtime-logs#).

**Reply by rajput-hemant** [reply]
Hi @amyegan,

Thank you for the response. I tried to follow your suggestion and access the Pro Trial option, but it seems to be unavailable for me. I've attached screenshots for reference. The first image is from the docs, showing the expected Pro Trial option, while the second image is what I see on my end.

![](https://vercel.com/_next/image?url=https%3A%2F%2Fassets.vercel.com%2Fimage%2Fupload%2Fv1694786173%2Fdocs-assets%2Fstatic%2Fdocs%2Fconcepts%2Fteams%2Fnew-team-dark.png&w=640&q=75&dpl=dpl_EWzVU1Ekv1HZWtvi4vLL3iG7Ufuy)

![1699701343](https://github.com/vercel/community/assets/68769346/e765f857-168f-4e0a-8c00-554d38f21840)


Any further guidance on accessing the Pro Trial or alternative solutions would be greatly appreciated.
